actions/rules/action_field_set_test.go

```diff
@@ -0,0 +1,96 @@
+package rules
+
+import (
+	"github.com/fabric8-services/fabric8-wit/convert"
+	"github.com/fabric8-services/fabric8-wit/resource"
+	"github.com/fabric8-services/fabric8-wit/workitem"
+	uuid "github.com/satori/go.uuid"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+
+	"github.com/fabric8-services/fabric8-wit/gormtestsupport"
+	tf "github.com/fabric8-services/fabric8-wit/test/testfixture"
+)
+
+func TestSuiteActionFieldSet(t *testing.T) {
+	resource.Require(t, resource.Database)
+	suite.Run(t, &ActionFieldSetSuite{DBTestSuite: gormtestsupport.NewDBTestSuite()})
+}
+
+type ActionFieldSetSuite struct {
+	suite.Suite
+	gormtestsupport.DBTestSuite
+}
+
+func (s *ActionFieldSetSuite) SetupSuite() {
+	s.DBTestSuite.SetupSuite()
+}
+
+func (s *ActionFieldSetSuite) SetupTest() {
+	s.DBTestSuite.SetupTest()
```

Comment:
No need to override this function. This is done for you automatically.

Reply:
Outdated. Already fixed.